app/services/employees/csv_parser.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Employees
+  class CsvParser < ApplicationService
+    attr_reader :attachment, :company, :approved_users
+
+    USER_PROPS = %i[email first_name last_name role phone].freeze
+
+    def initialize(attachment, company)
+      super()
+      @attachment = attachment
+      @company = company
+      @approved_users = []
+    end
+
+    def call
+      CSV.foreach(attachment, headers: true, header_converters: :symbol, col_sep: ';') do |row|
+        columns ||= row.headers.intersection(USER_PROPS)
```

Comment:
please, think about using active_import - it can help not to make multiple insert queries, but check that it will work with invites.